src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -29,7 +35,11 @@ function failed()
 }
 
 if [ ! -f "${TEMPLATE_PATH}" ]; then
-    failed "Must run from runner root or install is corrupt"
+    if [[ $IS_CUSTOM_TEMPLATE = 0 ]]; then
+        failed "Must run from runner root or install is corrupt"
+    else
+        failed "Invalid TEMPLATE_PATH env variable"
```

Comment:
Added distinction between custom template and default run error message

Reply:
should we add similar code to the `systemd.svc.sh.template` as well?

Reply:
Yes, I noticed. Running tests now on the repaired version. Do you want me to amend the commit? There is no reason why this one should exist.

Reply:
do whatever you want. 😄